app/src/main/java/com/ciandt/dragonfly/example/features/projectselection/ProjectSelectionActivity.kt

```diff
@@ -59,7 +63,16 @@ class ProjectSelectionActivity : BaseActivity(), ProjectSelectionContract.View {
         super.onResume()
         presenter.attachView(this)
 
-        if (projects.isEmpty()) {
+        if (firstTime) {
+            firstTime = false
```

Comment:
What happens if user goes to home and comes back to the app? Also, same question, but with Don't keep activities selected?